src/components/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export const Button = (props) => {
```

Comment:
[Optional]

We can use object destructing here instead since we are not directly using `props` in this component.

```diff
- export const Button = (props) => {
+ export const Button = ({wide, backgroundColor, onClick, className, value}) => {
```

Destructured variables can then be used like this (for example):

```diff
- const width = props.wide ? "50%" : "25%";
+ const width = wide ? "50%" : "25%";
```